### PR TITLE
[Compiler: Comment] add comment to initial value

### PR DESCRIPTION
**Pull request by Jim8y**
This pr adds comments to the initialvalueattribute, make it clear what it does.

### CODE REVIEW

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_StaticVar.cs [hunk 1]
```diff
@@ -1,17 +1,49 @@
...
+        /// </summary>
+        [InitialValue("024700db2e90d9f02c4f9fc862abaca92725f95b4fddcc8d7ffa538693ecf463a9", ContractParameterType.PublicKey)]
+        private static readonly ECPoint eCPoint = default;
+
+        /// <summary>
+        /// A static field of type UInt160 initialized with the InitialValue attribute. This allows for compile-time
+        /// initialization of blockchain-specific types like addresses, represented here as Hash160.
+        /// </summary>
+        [InitialValue("NXV7ZhHiyM1aHXwpVsRZC6BwNFP2jghXAq", ContractParameterType.Hash160)]
+        private static readonly UInt160 uInt160 = default;
```
Comment (shargon) [inline tests/Neo.Compiler.CSharp.TestContracts/Contract_StaticVar.cs:35]: Add UT?

Comment (Jim8y) [reply tests/Neo.Compiler.CSharp.TestContracts/Contract_StaticVar.cs:35]: sure, will do it later, but i copied this from other test contract, should already being tested. Will add it still if you think it is necessary

**Comment by Jim8y** [comment]
some weird issue happend in the unit test, 
8221706784426_ pic

It wont go into Push when I debug step by step, have no idea why.

**Comment by shargon** [comment]
> some weird issue happend in the unit test, 8221706784426_ pic
> 
> It wont go into Push when I debug step by step, have no idea why.

```
        static readonly BigInteger a2 = BigInteger.Parse("120");
        static readonly BigInteger a3 = BigInteger.Parse("3");
```

Require storage because those calls internally call `atoi`

**Review by shargon — APPROVED** [review]

**Comment by Jim8y** [comment]
> @Jim8y Require storage because those calls internally call `atoi`

Thank you shargon, will take me weeks without you.

**Pull request merged**